File: dagger-codegen/src/main/java/org/chelonix/dagger/codegen/introspection/InputValue.java
package org.chelonix.dagger.codegen.introspection;

public class InputValue {

    private String name;
    private String Description;
    private String defaultValue; // isDeprecated
    private TypeRef type;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String description) {
        Description = description;
    }

    public String getDefaultValue() {
        return defaultValue;
    }

    public void setDefaultValue(String defaultValue) {
        this.defaultValue = defaultValue;
    }

    public TypeRef getType() {
        return type;
    }

    public void setType(TypeRef type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return "InputValue{" +
                "name='" + name + '\'' +
                // ", Description='" + Description + '\'' +
                ", defaultValue='" + defaultValue + '\'' +
                ", type=" + type +
                '}';
    }
}

File: sdk/csharp/src/Dagger.SDK.CodeGen/Types/InputValue.cs
using System.Text.Json.Serialization;

namespace Dagger.SDK.CodeGen.Types;

public class InputValue
{
    [JsonPropertyName("name")]
    public required string Name { get; set; }

    [JsonPropertyName("description")]
    public required string Description { get; set; }

    [JsonPropertyName("type")]
    public required TypeRef Type { get; set; }

    [JsonPropertyName("defaultValue")]
    public string? DefaultValue { get; set; }

    [JsonPropertyName("directives")]
    public Directive[]? Directives { get; set; }
}
